internal/kibana/packages.go

```diff
@@ -27,9 +28,32 @@ func (c *Client) InstallPackage(pkg packages.PackageManifest) ([]packages.Asset,
 	return processResults("install", statusCode, respBody)
 }
 
+// InstallZipPackage installs the local zip package in Fleet.
+func (c *Client) InstallZipPackage(zipFile string) ([]packages.Asset, error) {
+	path := fmt.Sprintf("%s/epm/packages", FleetAPI)
+
+	body, err := os.Open(zipFile)
+	if err != nil {
+		return nil, errors.Wrap(err, "failed to read zip file")
+	}
```

Comment:
We should close this body.
```suggestion
	}
	defer body.Close()
```